fdbserver/workloads/Cycle.actor.cpp

```diff
@@ -102,13 +102,13 @@ struct CycleWorkload : TestWorkload {
 					try {
 						// Reverse next and next^2 node
 						Optional<Value> v = wait( tr.get( self->key(r) ) );
-						if (!v.present()) self->badRead("r", r, tr);
+						if (!v.present()) self->badRead("Keyr", r, tr);
```

Comment:
The convention is to use camel case as well, so to follow that you could do something like `"R"` or `"KeyR"`.

Reply:
Fixed. The updated commit uses `KeyR`